### PR TITLE
Add values for PR deploys, and start the workflow

### PR DESCRIPTION
**Pull request by ellie**
Beginnings of getting this out, testing the workflow is a little awkward as `act` isn't ideal for this and GitHub won't run feature branches OK

## Type of change
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by ellie** [comment]
Test comment

**Comment by ellie** [comment]
OK so it looks like GitHub won't be able to run this workflow without having it merged first. So I'm gonna do that, to make sure all this permissions stuff works OK

**Review by hazzadous — APPROVED** [review]

File: .github/workflows/pr-deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        with:
+          required-permission: write
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - run: |
+          echo A user with write permission has commented on PR $NUMBER
+        env:
+          NUMBER: ${{ github.event.issue.number }}
+        if: steps.check.outputs.has-permission
```
Comment (hazzadous) [inline .github/workflows/pr-deploy.yaml:29]: Seems to be some formatting/highlighting issue here 🤔

Comment (ellie) [reply .github/workflows/pr-deploy.yaml:29]: ahhh no missed it 😭 looks fine on my editor, but I can clean it up shortly if that turns out not to be the case

**Comment by ellie** [comment]
test PR comment

**Comment by ellie** [comment]
it did indeed 🙌

File: .github/workflows/pr-deploy.yaml (hunk 1 above)
Comment (ellie) [reply .github/workflows/pr-deploy.yaml:29]: just a highlighting problem :)

**Pull request merged**